src/clients/proxy.js

```diff
@@ -31,6 +48,14 @@ function init({ url, method, headers, body, log }) {
     delete options.headers.host;
     delete options.headers.authorization;
 
+    if (cpr) {
+      const copy = typeof body === "object" ? body : JSON.parse(body || {});
+      // attatch cprNumber to body according to url and method
+      const attachedCpr = attachCpr({ url, method, cpr });
+      // attach cpr to body
+      body = { ...copy, ...attachedCpr };
```

Comment:
Jeg tror ikke fletningen vil virke ordentligt for pincodeChange og guardian. Men vi skal have testene til at fange det